modules/installation-dns-user-infra.adoc

```diff
@@ -43,22 +43,22 @@ API calls can fail, and you cannot retrieve logs from Pods.
 ====
 
 |Routes
-|`*.apps.<cluster_name>.<base_domain>`
+|`*.apps.<cluster_name>.<base_domain>.`
```

Comment:
The update is correct, the `.` at the end of domain is required.

Reply:
yes, the name should have `.` at the end. but seems the `target` part of etcd SRV record `etcd-0.<cluster_name>.<base_domain>.` should not contain `.` at the end.
`_etcd-server-ssl._tcp.<cluster_name>.<base_domain>.  86400 IN    SRV 0        10     2380 etcd-0.<cluster_name>.<base_domain>.`

Reply:
yes, seems not required for target part.